src/BlockSync.ts

```diff
@@ -350,65 +395,67 @@ export class BlockSync {
+        const transactionFee = await this.extractTransactionFee(client, event, extrinsicEvents);
+        if (transactionFee > 0n) {
           const blockNumber = header.number.toNumber();
-          didChangeBiddings ||= await biddingFile.mutate(x => {
+          didChangeBiddings ||= await bidsFile.mutate(x => {
             if (x.lastBlockNumber >= blockNumber) {
               console.warn('Already processed cohort block', {
                 lastStored: x.lastBlockNumber,
                 blockNumber: blockNumber,
               });
               return false;
             }
-            x.fees += miningFee;
+            x.transactionFees += transactionFee;
           });
         }
       }
 
       if (phase.isFinalization && client.events.miningSlot.NewMiners.is(event)) {
         console.log('New miners event', event.data.toJSON());
         let hasWonSeats = false;
-        const [_startIndex, newMiners, _released, cohortId] = event.data;
-        await this.storage.biddingsFile(cohortId.toNumber()).mutate(x => {
-          x.seats = 0;
-          x.totalArgonsBid = 0n;
+        const [_startIndex, newMiners, _released, frameIdAtCohortActivation] = event.data;
+        await this.storage.bidsFile(frameIdAtCohortActivation.toNumber()).mutate(x => {
+          x.seatsWon = 0;
+          x.argonsBidTotal = 0n;
           x.subaccounts = [];
           x.lastBlockNumber = blockNumber;
 
-          let index = 0;
+          let bidPosition = 0;
           for (const miner of newMiners) {
             const address = miner.accountId.toHuman();
-            const bidAmount = miner.bid.toBigInt();
+            const argonsBid = miner.bid.toBigInt();
             const ourMiner = this.accountset.subAccountsByAddress[address];
+            
             if (ourMiner) {
               hasWonSeats = true;
-              x.seats += 1;
-              x.totalArgonsBid += bidAmount;
-              if (bidAmount > x.maxBidPerSeat) {
-                x.maxBidPerSeat = bidAmount;
-              }
+              x.seatsWon += 1;
+              x.argonsBidTotal += argonsBid;
               x.subaccounts.push({
-                subaccountIndex: ourMiner.index,
+                index: ourMiner.index,
                 address,
-                lastBidAtTick: miner.bidAtTick.toNumber(),
-                bidPlace: index,
+                lastBidAtTick: miner.bidAtTick?.toNumber(),
+                bidPosition,
+                argonsBid,
               });
             }
-            index++;
+            bidPosition++;
           }
         });
         await this.statusFile.mutate(x => {
+          x.bidsLastModifiedAt = new Date();
+          x.loadProgress = this.calculateProgress(this.currentTick, [this.oldestTick, this.latestTick]);
           if (hasWonSeats) {
             x.hasWonSeats = true;
           }
         });
       }
     }
+
     return didChangeBiddings;
   }
 
-  private async hasMiningFee(
+  private async extractTransactionFee(
```

Comment:
This rename seems to lose the idea that this will only return an amount if this was a mining transaction

Reply:
Fair. I can revert this name change.